Source/Core/VideoBackends/OGL/ProgramShaderCache.cpp

```diff
@@ -471,6 +471,9 @@ void ProgramShaderCache::Shutdown()
 	// store all shaders in cache on disk
 	if (g_ogl_config.bSupportsGLSLCache && !g_Config.bEnableShaderDebugging)
 	{
+		// Clear any prior error code
+		glGetError();
```

Comment:
Please move this into the loop, else an error within line 485 would prohibit two shaders from being dumped.